utils/swift_build_support/tests/products/test_swift.py

```diff
@@ -57,7 +57,8 @@ def setUp(self):
             benchmark_num_o_iterations=3,
             disable_guaranteed_normal_arguments=True,
             force_optimized_typechecker=False,
-            enable_stdlibcore_exclusivity_checking=False)
+            enable_stdlibcore_exclusivity_checking=False,
+            enable_experimental_differentiable_programming=False)
```

Comment:
btw, if you have an idea of how to make adding stuff here not needed, it would be nice (today this is a footgun as you saw).

Reply:
Noted, thanks for the info.